wavefront/server/modules/gold_module/gold_module/services/image_service.py

```
@@ -13,6 +13,13 @@ class ImageService:
     def __init__(self, cloud_service: CloudImageService):
         self.cloud_service = cloud_service
 
+    async def save_image(self, image_data: bytes, image_name: str):
+        validated_image_data = await self._validate_image(image_data)
+
+        bucket_name, file_path = await self.cloud_service.upload_image(
```

Ruff (0.15.0)

[warning] Unpacked variable `bucket_name` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)